src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -122,6 +122,27 @@ public void TestSupportedDiagnosticsMessageHelpLinkUri()
 
                 foreach (var descriptor in diagnosticAnalyzer.SupportedDiagnostics)
                 {
+                    // These don't come up in UI.
+                    if (descriptor.DefaultSeverity == DiagnosticSeverity.Hidden && descriptor.CustomTags.Contains(WellKnownDiagnosticTags.NotConfigurable))
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id is "RE0001") // Currently not documented. https://github.com/dotnet/roslyn/issues/48530
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id == "IDE0043") // Intentionally undocumented. It will be removed in favor of CA2241
+                    {
+                        continue;
+                    }
+
+                    if (descriptor.Id is "IDE1007" or "IDE1008")
```

Comment:
😕 I can't find any references to IDE1008

Reply:
https://github.com/dotnet/roslyn/pull/53234/files

Reply:
```suggestion
                    if (descriptor.Id == "IDE1007")
```